main.py

```diff
@@ -80,30 +76,42 @@ async def clear_react(message):
 	async def on_reaction_add(reaction, user):
 		global userInputMode
 		global category
-		# when a message has the "tick" emoji added, the user id is not the bot's and the channel is a dm 
-		if reaction.emoji == tick and user != client.user and isinstance(message.channel, discord.channel.DMChannel):
-			await reaction.message.channel.send("-----------------\nWhat category would you place your request under?\n\n"+one+" Moderation\n\n"+two+" Tournaments\n\n"+three+" Mentoring\n\n"+four+" Other\n-----------------")
-
-		# when a message has the "one" emoji added, the user id is not the bot's and the channel is a dm
-		if reaction.emoji == one and user != client.user and isinstance(message.channel, discord.channel.DMChannel):
+		
+		#  If the user id is the bot's or if the channel is not a dm, return
+		if(user == client.user or not isinstance(message.channel, discord.channel.DMChannel)):
```

Comment:
Ignore any messages thats sent by the bot OR not in DM. So every if function afterwards does not need to include it.